### ISSUE TITLE
Hamburger menu consistency across all pages.

### ISSUE THREAD

**Issue by PranavDani** [OurTechCommunity/catchup#114]
![image](https://user-images.githubusercontent.com/56108484/201331114-31fd22bd-91b6-4d48-a6fc-51e1fc833f3f.png)

![image](https://user-images.githubusercontent.com/56108484/201330782-d4ded916-dc10-401c-b3c7-15c87026b732.png)

Currently, About CatchUp acts as the only link between [summaries](https://catchup.ourtech.community/summary), the latest summary, and [OTC CatchUp](https://catchup.ourtech.community). 
So what I suggest is, instead of having the hamburger on the top right, we can move it to the top left across every single page so that it does not collide with the dark/light switch on summaries (This can also allow the addition of the switch across all the pages.). (Also having it on the right side feels **RIGHT** for some reason.)

- This will allow users to use the hamburger across all the pages.
- Maintains consistency across all the pages.
- Allows a proper replacement for the above-mentioned changes.

_Originally posted by @PranavDani in https://github.com/OurTechCommunity/catchup/issues/113#issuecomment-1311583033_

**Comment by HarshKapadia2** [comment]
Yeah it makes sense to me to have the hamburger on the top left corner pf the page. The reasons you gave are good in my opinion.

One more reason that I can think of is that in the future we might implement https://github.com/OurTechCommunity/website/issues/9 and we might add the theme-change button on the top right, just like with the OTC CatchUp summary pages.

Before we make this change though, maybe #55 should be implemented, so that we can make the change easily.
